Source/WebCore/css/parser/CSSParserFastPaths.cpp

```diff
@@ -751,6 +751,9 @@ bool CSSParserFastPaths::isValidKeywordPropertyAndValue(CSSPropertyID propertyId
     case CSSPropertyVisibility: // visible | hidden | collapse
         return valueID == CSSValueVisible || valueID == CSSValueHidden || valueID == CSSValueCollapse;
     case CSSPropertyAppearance: {
+        if (valueID == CSSValueMediaSlider || valueID == CSSValueMediaVolumeSlider || valueID == CSSValueMediaFullscreenVolumeSlider)
+            return isUASheetBehavior(context.mode);
```

Comment:
Can we move this logic to `isValueAllowedInMode` in `CSSParserIdioms`?

Reply:
As discussed offline, I prefer putting this logic here, along with other gating logic for appearance values